layouts/partials/bib/entry/extras.html

```diff
@@ -14,6 +14,9 @@
 {{- with .entry.doi -}}
   <span class="{{ $class }} pubref"><a href="https://doi.org/{{ . }}" target="_blank" rel="noopener">Release</a></span>
 {{- end -}}
+{{- with .entry.code -}}
+  <span class="{{ $class }} pubref"><a href="{{ . }}" target="_blank" rel="noopener">Code</a></span>
+{{- end -}}
```

Comment:
Move the whole `with`-block for the `.entry.code` before the one for
`.entry.doi`, i.e., move lines 17 through 19 to just above line 14.

I think it is better to keep the Release link the last one before the
Abstract and BibTeX buttons.